### ISSUE TITLE
Suggestion for a name change slight rebranding of the pangeo CMIP6 cloud holdings

### ISSUE THREAD

**Issue by jbusecke** [pangeo-data/pangeo-cmip6-cloud#42]
I recently gave a talk and read @balaji-gfdl s [paper](https://doi.org/10.5194/gmd-11-3659-2018) before that. 

If I understand this figure
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/14314623/169365364-800d9025-18b0-42bd-a8fc-f97408dee427.png">
right, what we are providing is in fact an 'ESGF Cache'.

I propose that we unify the naming of our 'cloud holdings' to something like this:

"Pangeo CMIP6 Cache"
"Pangeo CMIP6 Cloud Cache"

The “Cache” makes it clear that ESGF is still the main holding, and that we are leaving stuff as much ‘as is’ as possible.

Happy to make a PR and change this where appropriate, but I wanted to float this idea for comments/feedback first.

cc @AparnaRaveendran @rabernat @cisaacstern @agstephens

**Comment by cisaacstern** [comment]
If this refers to NetCDF holdings on the cloud, then I agree.

Regarding the WIP with Pangeo Forge, it seems that some type of language regarding the analysis-ready cloud-optimized nature of the cache would be an important clarification to provide in the naming scheme.

**Comment by jbusecke** [comment]
Good point. I initially actually suggested `Pangeo CMIP6 ARCO Cache` to @rabernat but thought that might be too long? 
Do you think this would work?

**Comment by cisaacstern** [comment]
I'll be interested to know what others think, but I wonder if a "cache" implies less transformation than ARCO requires? I recognize the value of aligning with Balaji's typology, but perhaps the Pangeo Forge activities are outside of the categories that framework defines?
